packages/client/lib/net/protocol/ethprotocol.ts

```diff
@@ -149,12 +151,19 @@ export class EthProtocol extends Protocol {
           const common = this.config.chainCommon
           // If this is a post merge block, we can still send chainTTD since it would still lead
           // to correct hardfork choice
-          return BlockHeader.fromValuesArray(
-            h,
-            difficulty > 0
-              ? { common, hardforkByBlockNumber: true }
-              : { common, hardforkByTTD: this.chainTTD }
-          )
+          try {
+            const header = BlockHeader.fromValuesArray(
+              h,
+              difficulty > 0
+                ? // TODO: Change this back before merging
+                  { common, hardforkByBlockNumber: true, skipConsensusFormatValidation: true }
+                : { common, hardforkByTTD: this.chainTTD }
+            )
+            return header
+          } catch (err: any) {
+            console.log(err)
```

Comment:
just marking this to cleanup while finalizing the PR

on seconds thought: should we add debug logging facility here (as a separate PR)? it would generally be helpful as generally errors get eaten when something is going wrong and we have to resort to these kind of tricks